### DISCUSSION TITLE
Use of LFIN with g1

### DISCUSSION THREAD

**Discussion by TripleFJeff** [gopro/labs#295] [Q&A]
In the page for generating a QR code to add an overlay to video, there is an option to use GPS (I assume GPS time vs camera time). If that is selected, "g0!" is substituted with "g1!MLFIN=0". Is the LFIN command really necessary? Ir seems like it should only be needed with the LAPS functions, which aren't even available on that page.

https://gopro.github.io/labs/control/overlays/

**Comment by dnewman-gpsw** [comment]
There was a bug in an earlier Labs (HERO8 most likely, maybe still there) that would randomly start the Labs overlays, the LFIN=0 patched that bug.
